### ISSUE TITLE
Spilt PRISM climatologies

### ISSUE THREAD

**Issue by corviday** [pacificclimate/climate-explorer-data-prep#140]
In order to make the PRISM data accessible to the new ncWMS, we need to replace the month+year climatologies with separate monthly and yearly climatologies.

- [ ] split climatologies with CDO
- [ ] update metadata
- [ ] index climatologies
- [ ] create new ensemble

**Comment by corviday** [comment]
- [ ]  associate into `downscaled_canada`
- [ ] update title and model descriptions after hearing from Faron

**Comment by corviday** [comment]
We're still waiting on some metadata from Faron, but it's think like contact info and data title that aren't required mechanically. I can update them when I have them without needing to reindex. So the remaining actions are lower priority.
